### PR TITLE
Dev/iwu/us vbs22 37 cambio y recuperacion de contraseña

### PR DESCRIPTION
**Pull request by ichivancrCenfotec**
Atención a los comentarios

### CODE REVIEW

File: src/app/interfaces/index.ts [hunk 1]
```diff
@@ -8,15 +8,21 @@ export interface IResponse<T> {
 }
 
 export interface IUser {
+ 
   id?: number;
   name?: string;
-  lastname?: string;
+  last_name?: string;
```
Comment (nefi-cenfo) [inline src/app/interfaces/index.ts:14]: Please consider to use camel case format.

File: src/app/pages/auth/login/login.component.html [hunk 2]
```diff
@@ -68,7 +70,6 @@ <h1 class="yeseva-one-regular font-size-4">Sign In</h1>
   </div>
   <footer class="footer">
     <div class="footer-line"></div>
-    <!-- <img src="../../../../assets/icons/Round_Voiage.webp" alt="Logo" class="footer-logo"> -->
```
Comment (nefi-cenfo) [inline src/app/pages/auth/login/login.component.html:71]: Can you delete the comented code?

File: src/app/pages/auth/login/login.component.scss [hunk 3]
```diff
@@ -142,28 +142,17 @@ a {
   justify-content: space-between;
   align-items: center;
   padding: 10px 20px;
-  // background: transparent;
```
Comment (nefi-cenfo) [inline src/app/pages/auth/login/login.component.scss:145]: Can you delete the comented code?

File: src/app/pages/auth/login/login.component.scss [hunk 4]
```diff
@@ -142,28 +142,17 @@ a {
...
-  // background: #775D68;
   background: #FADCD4;
   background-size: cover;
   background-position: center;
   margin-left: 0;
-  // flex-direction: column;
 }
 
-// .footer-line {
```
Comment (nefi-cenfo) [inline src/app/pages/auth/login/login.component.scss:154]: Can you delete the comented code?

**Review by nefi-cenfo — APPROVED** [review]

**Review by mur-tz — APPROVED** [review]
Approved

**Review by DieS06 — APPROVED** [review]

**Pull request merged**